deploy/mpi-operator.yaml

```diff
@@ -1,51 +1,11 @@
 # --------------------------------------------------
-# - Single configuration deployment YAML for MPI-Operator
+# - Configuration deployment YAML for MPI-Operator
```

Comment:
This file was originally purposed to provide one single file for MPI Operator installation. Maybe have a script to merge the generated CRD yaml with this one?